src/sass/user-profile.scss

```diff
@@ -12,6 +12,29 @@
   margin-bottom: 1.5em;
   padding: 0;
 
+  .mfa-message {
+    h2:first-of-type {
+      padding-top: 0.15em;
+    }
+
+    div + div {
+      padding-right: 0;
+      text-align: right;
+
+      button {
+        margin-right: 0;
+      }
+    }
+  }
+
+  .usa-alert {
```

Comment:
Can we make a class for these styles? If someone else adds an alert in the profile page, they might have to override these.